packages/website/pages/docs/reference/prover-market.mdx

```diff
@@ -0,0 +1,24 @@
+# Prover Market
```

Comment:
can we make this more explicit/declarative, perhaps: "Prover market registration page"